Mobile/starter_project_matador/lib/features/example/presentation/screen/signup.dart

```diff
@@ -0,0 +1,216 @@
+        title: 'Sign Up Page',
+        home: Scaffold(
+            backgroundColor: Colors.white,
+            body: SafeArea(
+                child: ListView(children: <Widget>[
+              SizedBox(
+                height: 130,
+                width: 234,
+                child: Image(
+                  height: 54,
+                  width: 141,
+                  image: NetworkImage(
+                      "https://media.licdn.com/dms/image/D4E16AQGS-26jpTdk2w/profile-displaybackgroundimage-shrink_200_800/0/1671689947030?e=2147483647&v=beta&t=6y-EPCeib7vMuOKrklKkiVnqCw8KEH0q6VfJovmBb9A"),
+                ),
+              ),
+              Container(
+                height: 76,
+                width: 295,
+                decoration: BoxDecoration(
+                  color: Color(0xff376AED),
+                  borderRadius: BorderRadius.only(
+                    topLeft: Radius.circular(28),
+                    topRight: Radius.circular(28),
+                  ),
+                  boxShadow: [
+                    BoxShadow(
+                      color: Color.fromRGBO(79, 91, 121, 0.1),
+                      blurRadius: 22,
+                      offset: Offset(0, 4),
+                    ),
+                  ],
+                ),
+                child: Row(
+                  mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+                  children: <Widget>[
+                    Text(
+                      'LOGIN',
+                      style: TextStyle(
+                        fontFamily: 'Urbanist',
+                        fontWeight: FontWeight.bold,
+                        fontSize: 18,
+                        letterSpacing: 1,
+                        color: Colors.white.withOpacity(0.25),
+                      ),
+                    ),
+                    Text(
+                      'SIGNUP',
+                      style: TextStyle(
+                        fontFamily: 'Urbanist',
+                        fontWeight: FontWeight.bold,
+                        fontSize: 18,
+                        letterSpacing: 1,
+                        color: Colors.white,
+                      ),
+                    ),
+                  ],
+                ),
+              ),
+              Expanded(
+                  child: Container(
+                width: double.infinity,
+                decoration: BoxDecoration(
+                  color: Colors.white,
+                  borderRadius: BorderRadius.only(
+                    topLeft: Radius.circular(28),
+                    topRight: Radius.circular(28),
+                  ),
+                  boxShadow: [
+                    BoxShadow(
+                      color: Color.fromRGBO(82, 130, 255, 0.09),
+                      blurRadius: 32,
+                      offset: Offset(0, -25),
+                    ),
+                  ],
+                ),
+                child: Padding(
+                  padding: const EdgeInsets.symmetric(
```

Comment:
Use the media query options to make sizes responsive. Make sure all the other fixed sizes are changed.